<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>telaResp</class>
 <widget class="QDialog" name="telaResp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="valX">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>valorX</string>
   </property>
  </widget>
  <widget class="QLabel" name="valY">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>valorY</string>
   </property>
  </widget>
  <widget class="QLabel" name="valFu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Função</string>
   </property>
  </widget>
  <widget class="QLabel" name="valN">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>valorN</string>
   </property>
  </widget>
  <widget class="QLabel" name="valH">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>valorH</string>
   </property>
  </widget>
  <widget class="QLabel" name="escPreditor">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Escolha do Preditor</string>
   </property>
  </widget>
  <widget class="QLabel" name="escMetodo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Posição da escolha do metodo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
